### DISCUSSION TITLE
Downloading Mods on Steam Games Through Whisky

### DISCUSSION THREAD

**Discussion by rileypokemon2** [Whisky-App/Whisky#649] [Q&A]
Trying to download the Bigger Lobby Mod for Lethal Company on Steam. I’ve tried downloading it natively on my Mac and copying the files into Lethal Company’s game files and it won’t work. Any ideas on how to download the mods through Whisky instead?

**Comment by PAOLOX00X** [comment]
Hello, if someone is not very good on this kind of thing, I made a YT video on how to install and mod lethal company on mac via whisky. This is the video
https://youtu.be/36E-Yklpp2E?si=ByNFz988ZEAOPMt4

**Reply by BLOKBUSTR** [reply]
I followed this tutorial on a 2020 M1 Mac Mini, and everything has worked flawlessly. Thank you!

**Reply by PAOLOX00X** [reply]
You are welcome
